Documentation/recommended-labels.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Recommended Labels
+weight: 2250
+indent: true
+---
+
+# Recommended Labels
+
+Recommended Labels are a common set of labels that allows tools to work interoperably, describing objects in a common manner that all tools can understand.
+
+## Labels
+
+`app.kubernetes.io/name`: Is the name of the binary running in a container.
+
+`app.kubernetes.io/instance`: A unique name identifying the instance of an application, it is the combination of daemonType and daemonID.
+
+`app.kubernetes.io/component`: It is populated as resource kind,
+for example: If `cephclusters` is the resource Kind so the targeted component will be `cephclusters.ceph.rook.io`. 
+
+`app.kubernetes.io/part-of`: It is the name of the resource.
+
+`app.kubernetes.io/managed-by`: `rook-ceph-operator` is the tool being used to manage the operation of an application
+
+`app.kubernetes.io/created-by`: `rook-ceph-operator` is the controller/user who created this resource
+
+`rook.io/operator-namespace`: The namespace in which rook-ceph operator is running.
+
+An Example of Recommended Labels on Ceph mon with ID=a will look like:
+```
+	app.kubernetes.io/name       : "ceph-mon"
+	app.kubernetes.io/instance   : "mon-a"
+	app.kubernetes.io/component  : "cephclusters"
+	app.kubernetes.io/part-of    : "rook-ceph-mon-a"
+	app.kubernetes.io/managed-by : "rook-ceph-operator"
+	app.kubernetes.io/created-by : "rook-ceph-operator"
+	rook.io/operator-namespace   : "rook-ceph"
+```
```

Comment:
Can add a note to help users tie together component, part-of, and instance.
```suggestion
```

A totally unique string for an application can be built up from (a) `app.kubernetes.io/component`, (b) `app.kubernetes.io/part-of`, (c) the resource's namespace, (d) `app.kubernetes.io/name`, and (e) `app.kubernetes.io/instance` fields. For the example above, we could join those fields with underscore connectors like this: `cephclusters.ceph.rook.io_rook-ceph_my-cluster_ceph-mon_a`. Note that this full spec can easily exceed the 64-character limit imposed on Kubernetes labels.
```